Examples/Io/Json/CMakeLists.txt

```diff
@@ -4,13 +4,14 @@ add_library(
   src/JsonMaterialWriter.cpp
   src/JsonSurfacesWriter.cpp
   src/JsonOptionsWriter.cpp
-  src/JsonDigitizationConfig.cpp)
+  src/JsonDigitizationConfig.cpp
+  src/JsonTGeoDetectorConfig.cpp)
 target_include_directories(
   ActsExamplesIoJson
   PUBLIC $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>)
 target_link_libraries(
   ActsExamplesIoJson
-  PUBLIC ActsCore ActsPluginJson ActsExamplesDigitization ActsExamplesFramework ActsExamplesMaterialMapping Boost::program_options)
+  PUBLIC ActsCore ActsPluginJson ActsPluginTGeo ActsExamplesDigitization ActsExamplesFramework ActsExamplesMaterialMapping Boost::program_options)
```

Comment:
With this, you can ONLY build the Json plugin if TGeo is configured. Is this guaranteed to be the case?

Reply:
I see. Probably not a good idea. Since I cannot forward declare the TGeoLayerBuilder config structs, I could move this back to the plugin json?

Reply:
Hm, but don't you have the same problem then? Maybe the way to go would be to conditionally add this to the `ActsExamplesIoJson` **if** the `TGeo` plugin is enabled. Do you think that makes things too complex?

Reply:
If I remember correctly, the include was enough then. You mean just if(ACTS_BUILD_PLUGIN_TGEO) ... or is there a more elegant way?

Reply:
That would be it. I don't immediately have any better ideas.